precompile/living-assets/contracts/LivingAssetsOwnership.sol

```diff
@@ -13,7 +13,7 @@ interface LivingAssets {
     function createCollection(
```

Comment:
Do we still want this interface? If so, should it not accept collection_id as input parameter?